datafusion/sqllogictest/test_files/math.slt

```diff
@@ -95,10 +95,10 @@ SELECT atan2(2.0, 1.0), atan2(-2.0, 1.0), atan2(2.0, -1.0), atan2(-2.0, -1.0), a
 1.107148717794 -1.107148717794 2.034443935796 -2.034443935796 NULL NULL NULL
 
 # nanvl
-query RRR
-SELECT nanvl(asin(10), 1.0), nanvl(1.0, 2.0), nanvl(asin(10), asin(10))
+query RRB
+SELECT nanvl(asin(10), 1.0), nanvl(1.0, 2.0), isnan(nanvl(asin(10), asin(10)))
```

Comment:
The sign of `NaN` returned by `asin(10)` seems different between operating systems.
So, I use `isnan` just to check if it's `NaN` or not.